src/System.Net.Security/tests/FunctionalTests/CertificateValidationClientServer.cs

```diff
@@ -69,19 +68,19 @@ public CertificateValidationClientServer()
                     Task clientAuthentication = sslClientStream.AuthenticateAsClientAsync(
                         serverName,
                         clientCerts,
```

Comment:
Assuming that CI runs them, all tests have passed. What was the prerequisite step that you would have expected here?